apps/base-docs/tutorials/docs/2_data-indexers-subquery.md

```diff
@@ -0,0 +1,373 @@
+---
+title: 'Indexing Data with SubQuery'
+slug: /data-indexers-subquery
+description: A tutorial explaining SubQuery and providing swift guidance on configuring a project
+author: subquery
+keywords:
+  [
+    indexers,
+    RPC,
+    API,
+    graphql,
+    frontend,
+    smart contract development,
+    EVM,
+    Base,
+    Base network,
+    Base node providers,
+    Base providers,
+    blockchain development,
+    dApps,
+    smart contracts,
+    blockchain API
+  ]
+tags: ['smart contracts', 'data indexing']
+difficulty: beginner
+hide_table_of_contents: false
+displayed_sidebar: null
+---
+
+This tutorial will guide you through the process of configuring a SubQuery indexer for Base.
+
+---
+
+## Objectives
+
+- Introduce the concept of indexers
+- Configure SubQuery indexers to efficiently retrieve data from a smart contract
+
+The goal is to index the all the claims from the [Bridge to Base NFT contract][1] on [Base Mainnet](https://docs.base.org/using-base/).
+
+---
+
+:::info
+
+The final code of this project can be found [here](https://github.com/subquery/ethereum-subql-starter/tree/main/Base/base-nft).
+
+:::
+
+---
+
+The contents presented in this tutorial were discussed previously during the workshop. You can follow along to receive detailed audio commentary on the setup's intricacies:
+
+<iframe width="1317" height="741" src="https://www.youtube.com/embed/8N5h3SI8E-A" title="Base x SubQuery Indexing Workshop" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```

Comment:
Please do not include an iFrame and link to the video above when referencing "a workshop"